librecad/src/lib/gui/rs_linetypepattern.cpp

```diff
@@ -72,4 +72,4 @@ const RS_LineTypePattern RS_LineTypePattern::patternBorderLine2{{6.0, -3.0, 6.0,
 const RS_LineTypePattern RS_LineTypePattern::patternBorderLineX2{{24.0, -8.0, 24.0, -8., 0.2, -8.}};
 
 const RS_LineTypePattern RS_LineTypePattern::patternBlockLine{{0.5, -0.5}};
-const RS_LineTypePattern RS_LineTypePattern::patternSelected{10.0};
+const RS_LineTypePattern RS_LineTypePattern::patternSelected={10.0};
```

Comment:
The first assignment already worked. When you are in doubt about the correct or preferred syntax, don't hesitate to ask.
The preferred place for code related discussions is on our [Zulip chat](https://librecad.zulipchat.com)